examples/example_app.py

```diff
@@ -0,0 +1,43 @@
+from flask import Flask, render_template_string
```

Comment:
can you please include licence as it's specified in `flask_breadcrumbs/__init__.py` with current year and rename the file to `examples/app.py`.